reconcile/external_resources/manager.py

```diff
@@ -161,19 +161,20 @@ def _get_desired_objects_reconciliations(self) -> set[Reconciliation]:
                 continue
             module = self.module_inventory.get_from_spec(spec)
             try:
-                resource = self._build_external_resource(spec, self.er_inventory)
+                resource = self._build_external_resource(spec)
             except ExternalResourceValidationError as e:
                 self.errors[key] = e
                 continue
 
             reconciliation = Reconciliation(
                 key=key,
                 resource_hash=resource.hash(),
-                input=self._serialize_resource_input(resource),
+                input=resource.json(),
```

Comment:
Looks like `_serialize_resource_input` isn't in use anymore. Please remove it.

Reply:
It's already removed.